src/rippled/lib/rippled.js

```diff
@@ -76,63 +76,64 @@ const getLedger = (rippledSocket, parameters) => {
 }
 
 // get ledger_entry
-const getLedgerEntry = (rippledSocket, { index }) => {
+const getLedgerEntry = (rippledSocket, index, includeDeleted = false) => {
   const request = {
     command: 'ledger_entry',
     index,
     ledger_index: 'validated',
+    include_deleted: includeDeleted,
   }
 
   return query(rippledSocket, request).then((resp) => {
-    if (resp.error_message === 'entryNotFound') {
+    if (resp.error === 'entryNotFound') {
       throw new Error('ledger entry not found', 404)
     }
 
-    if (resp.error_message === 'invalidParams') {
+    if (resp.error === 'invalidParams') {
       throw new Error('invalidParams for ledger_entry', 404)
     }
 
-    if (resp.error_message === 'lgrNotFound') {
+    if (resp.error === 'lgrNotFound') {
       throw new Error('invalid ledger index/hash', 400)
     }
```

Comment:
This RPC command always fetches the last `validated` ledger. This error cannot possibly occur.